xbmc/interfaces/legacy/InfoTagMusic.h

```diff
@@ -406,6 +406,96 @@ namespace XBMCAddon
 #else
       String getLyrics();
 #endif
+
+#ifdef DOXYGEN_SHOULD_USE_THIS
+      ///
+      /// \ingroup python_InfoTagMusic
+      /// @brief \python_func{ getMusicBrainzTrackID() }
+      ///-----------------------------------------------------------------------
+      /// Returns the MusicBrainz Recording ID from music info tag (if present).
+      ///
+      /// @return [string] MusicBrainz Recording ID
+      ///
+      ///
+      ///-----------------------------------------------------------------------
+      /// @python_v18 New function added.
+      ///
+      getMusicBrainzTrackID();
+#else
+      String getMusicBrainzTrackID();
+#endif
+
+#ifdef DOXYGEN_SHOULD_USE_THIS
+      ///
+      /// \ingroup python_InfoTagMusic
+      /// @brief \python_func{ getMusicBrainzArtistID() }
+      ///-----------------------------------------------------------------------
+      /// Returns the MusicBrainz Artist IDs from music info tag (if present).
+      ///
+      /// @return [list] MusicBrainz Artist IDs
+      ///
+      ///
+      ///-----------------------------------------------------------------------
+      /// @python_v18 New function added.
+      ///
+      getMusicBrainzArtistID();
+#else
+      std::vector<String> getMusicBrainzArtistID();
+#endif
+
+#ifdef DOXYGEN_SHOULD_USE_THIS
+      ///
+      /// \ingroup python_InfoTagMusic
+      /// @brief \python_func{ getMusicBrainzAlbumID() }
+      ///-----------------------------------------------------------------------
+      /// Returns the MusicBrainz Release ID from music info tag (if present).
+      ///
+      /// @return [string] MusicBrainz Release ID
+      /// @python_v18 New function added.
```

Comment:
About this better to move down a bit like on the other places. Otherwise it becomes by doxygen added as text to `@return`

Reply:
Also forgot to ask about python_v18, shouldn't that be python_v19?

Reply:
Oops. Copy/paste gone wrong. Fixed now. Thanks!

I didn’t see any `@python_v19` when I did the updates earlier… but I also realised I hadn’t `fetch`’d since I last worked on it, so yes, I think _v19 is more appropriate, so I fixed those too.